<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Research</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/bootstrap.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Cara E. Brook</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="research_interests.html">Research</a>
</li>
<li>
  <a href="publications.html">Publications</a>
</li>
<li>
  <a href="about.html">Bio</a>
</li>
<li>
  <a href="team.html">Team</a>
</li>
<li>
  <a href="opportunities.html">Opportunities</a>
</li>
<li>
  <a href="https://carabrook.github.io/blog">Popular Writing</a>
</li>
<li>
  <a href="https://carabrook.github.io/E2M2">E2M2</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore"><strong>Research</strong></h1>

</div>


<style type="text/css">
h1.title {
  font-size: 31px;
  text-align: left;
  color:black;
}
</style>
<div style="text-align: left; font-size: 1.4em;">
<p><strong>Bats as reservoirs for emerging viral zoonoses</strong></p>
</div>
<img src="flying_bat.png" alt="bat" style="height: 250px; padding-right: 10px;" align="left">
<div style="text-align: left; font-size: 1.2em; padding-right=100px">
<p>Bats are reservoir hosts for the world’s most virulent emerging viral zoonoses, including Hendra and Nipah henipaviruses, Ebola and Marburg filoviruses, and SARS, MERS, and SARS-CoV-2 coronaviruses. Bats host these viruses without experiencing extensive morbidity or mortality, leading researchers to ask whether bats might be uniquely adapted for their roles as pathogen hosts. Accumulating molecular evidence links the evolution of flight to bat viral tolerance: unique anti-inflammatory pathways in bat systems enable them to avoid oxidative damage accrued during the metabolically expensive process of flight, with cascading consequences for bat longevity and resilience to intracellular infections (<a href="https://science.sciencemag.org/content/339/6118/456">Zhang et al. 2013</a>, <a href="https://www.sciencedirect.com/science/article/pii/S0092867418304021">Pavlovich et al. 2018</a>, <a href="https://www.sciencedirect.com/science/article/pii/S1931312818300416">Xie et al. 2018</a>). I bridge field ecology, cellular immunology, and quantitative epidemiology to investigate both <strong>population-level</strong> and <strong>within-host</strong> questions related to this research theme. The fitting of dynamic models to time series data represents an underlying theme in my work. I derive data from a variety of laboratory and field sources, with a particular focus on viral infections in Madagascar fruit bats.</p>
</div>
<hr />
<div style="text-align: left; font-size: 1em;">
<p><strong>Brook CE</strong> and Dobson AP. 2015. Bats as ‘special’ reservoirs for emerging zoonotic pathogens. Trends in Microbiology. doi: <a href="https://doi.org/10.1016/j.tim.2014.12.004">10.1016/j.tim.2014.12.004</a></p>
<p>Guth S, Visher E, Boots M, and <strong>Brook CE.</strong> 2019. Host phylogenetic distance drives trends in virus virulence and transmissibility across the animal-human interface. Philosophical Transactions of the Royal Society. 374(1782): 20190296. doi: <a href="https://doi.org/10.1098/rstb.2019.0296">10.1098/rstb.2019.0296</a>.</p>
</div>
<hr />
<div style="text-align: left; font-size: 1.4em;">
<p><strong>Population-level: How do bat viruses persist?</strong></p>
</div>
<img src="bat_pop.png" alt="bat_cell" style="height: 300px; padding-right: 10px;" align="right">
<div style="text-align: left; font-size: 1.2em; padding-right=100px">
<p>Classic epidemiological modeling explores the dynamics of perfectly immunizing childhood infections, under which hosts are born susceptible (S), become infectious (I), and recover (R) via immune responses retained for life. In the simplest example, infections persist maintained in a population by constant re-supply of susceptible births. Many bat species reproduce in annual or biannual birth pulses, meaning that susceptible re-supply is restricted within a year. Such a system suggests that bat viruses may be governed by more nuanced dynamics than standard SIR–be they longer infectious periods, latent periods for persistent infections, or periodic waning immunity (<a href="https://doi.org/10.1371/journal.pntd.0004796">Plowright et al. 2016</a>). To elucidate these dynamics, I fit candidate mechanistic models to age-structured prevalence and seroprevalence data, collected monthly with the help of the inimitable <a href="team.html">Ekipa Fanihy</a>. We find support for assumptions of waning antobody-mediated immunity but data are not yet finescale enough to distinguish between hypotheses of waning immunity and reinfection vs. persistent infection and periodic viral shedding. As part of an <a href="http://grantome.com/grant/NIH/R01-AI129822-01">NIH</a>- and <a href="https://www.bat1health.org/">DARPA</a>-funded research initiative, we are longitudinally sampling Malagasy fruit bat populations to construct the time series needed to parse our hypotheses.</p>
<hr />
</div>
<div style="text-align: left; font-size: 1em;">
<p><strong>Brook CE</strong>, Ranaivoson HC, Broder CC, Cunningham AA, Héraud JM, Peel AJ, Gibson L, Wood JLN, Metcalf CJE*, and Dobson AP*. 2019. Disentangling serology to elucidate henipa- and filovirus transmission in Madagascar fruit bats. Journal of Animal Ecology. doi: <a href="https://doi.org/10.1111/1365-2656.12985">10.1111/1365-2656.12985</a>. *=equal senior contributions.</p>
</div>
<hr />
<div style="text-align: left; font-size: 1.4em;">
<p><strong>Within-host: Why are bat viruses so virulent?</strong></p>
</div>
<img src="PaKi_EBOV_induced_10K_200hrs_CELL.gif" alt="bat_cell" style="height: 300px; padding-right: 10px;" align="left">
<div style="text-align: left; font-size: 1.2em; padding-right=100px">
<p>Recent work has demonstrated that some species of Old World Fruit Bat constitutively express the cytokine, Interferon-alpha, meaning that they have perpetually-primed antiviral immune systems (<a href="http://www.pnas.org/content/113/10/2696">Zhou et al. 2016</a>). This unique immunity likely plays a role in bats’ capacities to resist or tolerate viral infections that cause extreme virulence in other mammalian hosts. In collaboration with Albert Einstein College of Medicine, I studied the dynamics of bat viruses <em>in vitro</em>, modeling the spread of infection across a monolayer of bat cells under differing assumptions of innate antiviral immune status. We found that infections established under robust antiviral conditions in bat cell lines generated parameter estimates for within-host viral propagation rates a full order of magnitude higher than those estimated in the absent of an effect for immunity. Such findings suggest that viruses evolved in highly antiviral immune systems–like those demonstrated for bats–may achieve elevated replication rates likely to cause extreme pathology upon emergence into a host lacking constitutive immune defenses. As a Miller Fellow, I am further exploring the impact of a generalist anti-viral immune systems on the evolution of viral virulence, using a theoretical, adaptive dynamics approach.</p>
</div>
<hr />
<div style="text-align: left; font-size: 1em;">
<p><strong>Brook CE</strong>, Boots M, Chandran KC, Dobson AP, Drosten C, Graham AL, Grenfell BT, Müller MA, Ng M, Wang LF, and van Leeuwen A. 2020. Accelerated viral dynamics in bat cell lines, with implications for zoonotic emergence. eLife. doi: <a href="https://elifesciences.org/articles/48401">10.7554/eLife.48401</a>.</p>
</div>
<div style="text-align: left; font-size: 1em;">
<p><em>Watch the the National Science Foundation’s ‘4 Awesome Discoveries’ video summary of our eLife paper here:</em></p>
</div>
<iframe width="350" height="280" src="https://www.youtube.com/embed/y5_g_pjf0eE" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen>
</iframe>
<hr />
<div style="text-align: left; font-size: 1.4em; padding-right=80px">
<p><strong>Cross-scale: Metagenomic Next Generation Sequencing (mNGS) to identify zoonotic etiology of undiagnosed human fevers in Madagascar</strong></p>
</div>
<img src="christian_CZB.jpg" alt="chris_czb" style="height: 300px; padding-left: 10px;" align="left">
<div style="text-align: left; font-size: 1.2em; padding-right=150px">
<p>I am a recent recipient of a <a href="https://gcgh.grandchallenges.org/grant/metagenomics-and-etiology-zoonotic-disease-deciphering-bat-human-viral-transmission-madagascar">Bill &amp; Melinda Gates Foundation Grand Challenges Explorations grant</a> for ‘Metagenomic Next Generation Sequencing for Pathogen Identification in Low-Income Countries.’ In part with this award, Malagasy colleagues from the <a href="http://www.pasteur.mg/">Institut Pasteur of Madagascar</a> (IPM) and myself trained in library preparation and metagenomic Next Generation Sequencing at Chan Zuckerberg Biohub (San Francisco, CA) in July 2019. In March 2020, we received an Illumina sequencing machine at IPM and are currently undertaking sequencing of bat excretia from our longitudinal study concurrently with sequencing of samples from undiagnosed febrile human patients reporting to public hospitals across Madagascar. Our goal is to describe the landscape of potential viral zoonoses in biological samples from wild bats and identify any bat-derived zoonotic viruses responsible for human fever in Madagascar.</p>
</div>
<hr />
<div style="text-align: left; font-size: 1.4em; padding-right=80px">
<p><strong>Phylodynamics: Using NGS to inform COVID-19 outbreak response in Madagascar</strong></p>
</div>
<img src="MadaSARSCoV2.png" alt="Mada_SC2" style="height: 300px; padding-left: 10px;" align="left">
<div style="text-align: left; font-size: 1.2em; padding-right=500px">
<p>Madagascar reported its first case of COVID-19 on March 20, 2020 and has since diagnosed several more than 16,000 cases of COVID-19 from just under 80,000 tests (daily updated data available at <a href="https://www.covid19mg.org/dashboard_EN.html">COVID-19 Madagascar Dashboard</a>. I am a recent recipient of a grant from UC Berkeley’s <a href="https://innovativegenomics.org/projects/covid-19-in-madagascar-next-generation-sequencing/">Innovative Genomics Institute</a> to use the NGS tools outlined above at Institut Pasteur of Madagascar to sequence SARS-CoV-2 in country. Our main goals are to use these sequence data to: [1] Estimate timing of the origin of the COVID-19 epidemic in Madagascar, [2] Estimate the underreporting rate of qPCR-based positive case counts in Madagascar, and [3] Decipher patterns of cross-island connectivity and community spread across the island. We have recently released the two first sequences of SARS-COV-2 from human cases in Madagascar to GISAID, and they are available for viewing on <a href="https://nextstrain.org/sars-cov-2/">Nextstrain</a>. Stay tuned for more mechanistic insights as we expand our sequencing efforts soon!</p>
</div>
<hr />
<div style="text-align: left; font-size: 1.4em;">
<p><strong>Conservation: What population trends do we predict for threatened Madagascar fruit bats?</strong></p>
</div>
<img src="Fig1_AgeFreq.png" alt="bat_cell" style="height: 300px; padding-right: 10px;" align="right">
<div style="text-align: left; font-size: 1.2em; padding-right=100px">
<p>In part with our longitudinal field studies, we collect lower left premolar teeth from live-caught Malagasy <em>Pteropus rufus</em> and <em>Eidolon dupreanum</em> under anesthesia. Fruit bats put down annual layers of a tissue called <em>cementum</em> in their teeth; thus, from slicing, staining, and examining teeth in cross section, we can learn a fruit bat’s age. In addition to facilitating analyses of transmission dynamics for bat-borne pathogens, age data allow us to build age-frequency curves from which we can infer annual mortality rates and make population projections, using Leslie matrix modeling approaches. To date, our studies suggest that the Madagascan flying fox, <em>Pteropus rufus,</em> is in serious population decline. We are currently working with a local conservation NGO, <a href="https://www.madagasikara-voakajy.org/">Madigasikara Voakajy</a>, to develop conservation-oriented intervention plans for this species.</p>
</div>
<hr />
<div style="text-align: left; font-size: 1em;">
<p><strong>Brook CE</strong>, Ranaivoson HC, Andriafidison D, Ralisata M, Razafimanahaka J, Héraud JM, Dobson AP, and Metcalf CJE. 2019. Population trends for two Malagasy fruit bats. Biological Conservation. 234: 165-171. doi: <a href="https://doi.org/10.1016/j.biocon.2019.03.032">10.1016/j.biocon.2019.03.032</a>.</p>
</div>
<hr />




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
